src/transformers/models/instructblip/modeling_instructblip.py

```diff
@@ -1124,19 +1124,19 @@ def get_extended_attention_mask(
 
     def forward(
         self,
-        input_ids,
-        attention_mask=None,
-        position_ids=None,
-        query_embeds=None,
-        head_mask=None,
-        encoder_hidden_states=None,
-        encoder_attention_mask=None,
-        past_key_values=None,
-        use_cache=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-    ):
+        input_ids: torch.LongTensor,
+        attention_mask: Optional[torch.FloatTensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
+        query_embeds: Optional[torch.Tensor] = None,
```

Comment:
I was unsure of the `dtype`  (I couldn't find any reference) so I left the generic `torch.Tensor`